sql/backfill.go

```diff
@@ -95,22 +96,26 @@ func (p *planner) backfillBatch(b *client.Batch, tableName *parser.QualifiedName
 	}
 
 	if len(droppedColumnDescs) > 0 {
-		var updateExprs parser.UpdateExprs
-		for _, droppedColumnDesc := range droppedColumnDescs {
-			updateExprs = append(updateExprs, &parser.UpdateExpr{
-				Names: parser.QualifiedNames{&parser.QualifiedName{Base: parser.Name(droppedColumnDesc.Name)}},
-				Expr:  parser.DNull,
-			})
-		}
-
-		// Run `UPDATE <table> SET col1 = NULL, col2 = NULL, ...` to clear
-		// the data stored in the columns being dropped.
-		if _, err := p.Update(&parser.Update{
-			Table: table,
-			Exprs: updateExprs,
-		}); err != nil {
+		// Run a scan across the table using the primary key.
+		start := roachpb.Key(MakeIndexKeyPrefix(newTableDesc.ID, newTableDesc.PrimaryIndex.ID))
+		// Use a different batch to perform the scan.
+		batch := &client.Batch{}
+		batch.Scan(start, start.PrefixEnd(), 0)
```

Comment:
This potentially loads a lot of key/value pairs. I think I have an issue to fix `scanNode` to work in batches. Add a TODO to fix this code as well.